src/app/submission/sections/sections.service.ts

```diff
@@ -328,6 +328,22 @@ export class SectionsService {
       distinctUntilChanged());
   }
 
+  /**
+   * Check if given section id is of a given section type
+   * @param submissionId
+   * @param sectionId
+   * @param sectionType
+   */
+  public isSectionType(submissionId: string, sectionId: string, sectionType: SectionsType): Observable<boolean> {
```

Comment:
Could we add specs to `sections.service.spec.ts` for this new method?

Reply:
Done.

Reply:
Thanks for the quick response @ybnd !  The new tests look good to me.  So, as soon as @atarix83 is able to glance at this and approve, this can be merged!